app/lego.py

```diff
@@ -233,6 +256,8 @@ def startLego(self):
                     # Reload the tags config file
                     nfc.load_tags()
                     tags = nfc.tags
+                    mp3_dir = tags['mp3_dir']
```

Comment:
Let's check if mp3_dir is set, otherwise set a default. Replace line 259 with these:
```
try:
    mp3_dir = tags['mp3_dir'] + '/'
except Exception:
    mp3_dir = os.path.dirname(os.path.abspath(__file__)) + '/../music/'
```

Reply:
ok done